payments.md

```diff
@@ -226,24 +228,28 @@ To accomplish this, we introduce the Payment Reconciliation Protocol.
 
 This is a libp2p service run by all participants wanting to participate in payment reconciliation. When Alice has a set of payments from Bob that she is ready to cash out, Alice can send a `ReconcileRequest` to Bob, containing the following information:
 
-```go
+```sh
 type ReconcileRequest struct {
-	Vouchers []Vouchers
-
-	ReqVal TokenAmount
+	vouchers [Vouchers]
+	reqVal TokenAmount
 }
 ```
 
 The Vouchers should all be valid vouchers from Bob to Alice, on the same payment channel, and they should all be ready to be cashed in. `ReqVal` is a token amount less than or equal to the sum of all the values in the given vouchers. Generally, this value will be between the total sum of the vouchers, and that total sum minus the fees it would cost to submit them all to the chain.
 
 Bob receives this request, and checks that all the fields are correct, and then ensures that the difference between ReqVal and the vouchers sum is sufficient (this is a parameter that the client can set).  Then, he sends back a response which either contains the requested voucher, or an error status and message.
 
-```go
+```sh
 type ReconcileResponse struct {
-	Combined Voucher
+	combined Voucher
+	status  Status
+	message String optional
```

Comment:
Modifiers like 'optional' come before the TypeTerm.

Reply:
that is really confusing :(

Reply:
fixed